simulariumio/writers/binary_values.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
```

Comment:
`BinaryValues` stores values to be saved in binary and their corresponding format strings